src/pages/individual/Individual.jsx

```diff
@@ -192,7 +222,11 @@ export default function Individual() {
             )}
             assets={fakeAssets}
           />
-          <MetadataCard editable metadata={metadata} />
+          <MetadataCard
+            editable
```

Comment:
@colinwkingen is there a way to tell whether the currently logged in user is able to edit an individual's metadata?